util/db/migrations/20210915085630-update-expiry-date-column.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+if (process.env.NODE_ENV === 'production') {
+  module.exports = {
+    up: async (queryInterface, Sequelize) => {
+      // Grab the registrations as an array of objects.
+      const resultsArray = await queryInterface.sequelize.query('SELECT * FROM traps."Registrations";', {
+        type: Sequelize.QueryTypes.SELECT
+      });
+
+      // Loop through the results and add an expiry date value, calculated from the createdAt field.
+      for (const result of resultsArray) {
+        // Get the current date.
+        result.expiryDate = new Date();
+        // Add 4 years.
+        result.expiryDate.setFullYear(result.expiryDate.getFullYear() + 4);
+        // Set the month to December and the date to the 31st.
+        result.expiryDate.setMonth(11, 31);
```

Comment:
This will work for all the traps registered in 2021, but will erroneously give the 2020 registrants an extra year. You're best to use the createdAt field for the migration's 'today' value.